requirements/i18n.txt

```diff
@@ -1 +1,2 @@
 django>=1.11.2,<2.0
```

Comment:
This package, credentials-themes, already requires Django (https://github.com/edx/credentials-themes/blob/master/setup.py#L47), as does edx-i18n-tools. There is no need to specify this requirement again.